conf/test_scores.config

```diff
@@ -24,7 +24,7 @@ params {
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
     // Input data
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/nanostring/samplesheets/samplesheet_test.csv'
-    gene_score_yaml = 'https://raw.githubusercontent.com/nf-core/test-datasets/nanostring/gene_score_test.yaml'
+    input           = 'https://raw.githubusercontent.com/nf-core/test-datasets/nanostring/samplesheets/samplesheet_test.csv'
```

Comment:
You can add the same `params.pipelines_testdata_base_path`  here, like you did in the line after this one.